examples/http.glu

```diff
@@ -123,8 +123,8 @@ let catch_error action catch : Handler a -> (String -> Handler a) -> Handler a =
 /// Takes a `Handler` and a `Request` tries to process the request
 let handle handler state : Handler Response -> HttpState -> IO Response =
     let not_found _ _ =
-        pure { status = NotFound, body = "Page not found" }
-    handler (\response _ -> pure response) not_found state
+        wrap { status = NotFound, body = "Page not found" }
+    handler (\response _ -> wrap response) not_found state
```

Comment:
Tentatively I'd agree. `pure` only makes any sense when looking at it as "embedding a pure" value in an impure calculation. But that is not how my thought process goes really as the "pure value" is often derived from a previous impure calculation (as is the case here with `response`) which makes it not feel very "pure". `pure` does have precendence from Haskell though (as do `return` but that is confusing for other reasons).

Let me digest it a bit before merging!

Also, hello again @brendanzab !